src/oatpp/web/server/HttpProcessor.cpp

```diff
@@ -219,13 +219,17 @@ HttpProcessor::ConnectionState HttpProcessor::processNextRequest(ProcessingResou
 // Task
 
 HttpProcessor::Task::Task(const std::shared_ptr<Components>& components,
-                          const std::shared_ptr<oatpp::data::stream::IOStream>& connection)
+                          const std::shared_ptr<oatpp::data::stream::IOStream>& connection,
+                          const std::shared_ptr<std::atomic_ulong>& taskCounter)
   : m_components(components)
   , m_connection(connection)
+  , m_counter(taskCounter)
 {}
 
 void HttpProcessor::Task::run(){
 
+  (*m_counter)++;
```

Comment:
Should be incremented in the constructor of the `Task` or else,
someone can successfully call `stop()` before the thread started and `run()` was called.